news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md

```diff
@@ -33,6 +33,7 @@ Go download and install this RC version as a test store, either on your machine
 
 **Core:**
   - [Module Faceted search version has been bumped to 3.0.5](https://github.com/PrestaShop/PrestaShop/pull/14284) which contains interesting bug fixes
+  - Watermark and Product comments modules have been integrated and can now be installed from the Module Catalog page in the back office ;
```

Comment:
Add also : "- Tips and Updates module (psaddonsconnect) has been updated to the 2.0.0 version"

Reply:
done